test/config/integration/server_header.json

```diff
@@ -0,0 +1,181 @@
+{
```

Comment:
Two questions: Should this be YAML? Also, instead of a entire new base config, should we do this as a config modifier? There are other examples of this in the new integration tests. Maybe this is required because of the new param map stuff?